frontend/components/site/SiteSettings/AdvancedSiteSettings.jsx

```diff
@@ -117,16 +118,30 @@ export const AdvancedSiteSettings = ({
       Save advanced settings
     </button>
 
-    <AlertBanner
-      status="warning"
-      header="Danger zone"
-      message="Delete this site and its previews from Federalist, for all users?"
-      alertRole={false}
-    >
-      <button className="usa-button usa-button-red" onClick={onDelete}>
-        Delete
-      </button>
-    </AlertBanner>
+    <div className="well">
+      {/* DELETE SITE */}
+      <fieldset>
+        <legend>Delete Site</legend>
+        <p className="well-text">
+          Deleting a Federalist site removes the published site from our servers and
+          disconnects the Federalist admin interface for all users. This will bring the
+          entire site offline and make it inaccessible for users. <i>Trying to remove a site
+          from your list of Federalist sites? Go to the
+          <a href={`/sites/${siteId}/users`}>Collaborators page</a> and remove yourself.</i>
```

Comment:
Little bit of weirdness here thanks to [our friends at Facebook](https://github.com/facebook/react/issues/1643). Also, we may want to lowercase the 'C'.

![image](https://user-images.githubusercontent.com/963654/41788987-3c5605a4-7613-11e8-95b2-139fe1f78fcb.png)

Reply:
@amirbey - this is why I wanted the {' '} - I can re-add it or do you want to?